### DISCUSSION TITLE
Open Safari Tab Groups

### DISCUSSION THREAD

**Discussion by DiegoAz** [ttscoff/bunch-docs#286] [Ideas]
It would be great if Safari Tab Groups could be opened

**Comment by LuiFerreiraD** [comment]
Indeed! Just downloaded Bunch and this seems to be an amazing idea! Since Safari has the "Tab Groups" it would be nicer to just select one, this way it can dynamically change if we want/need.

**Comment by ttscoff** [comment]
So the question is —and I haven’t had time to look into this yet —are tab groups exposed through AppleScript at all, or failing that, can you use keyboard shortcuts? If neither are true, then there’s nothing Bunch can do. There’s no API for Safari interaction other than AppleScript.

- Brett

**Reply by DiegoAz** [reply]
I was searching on this topic and:
- regarding AppleScript [I found out that there is no way](https://www.reddit.com/r/applescript/comments/qf52db/open_group_tabs_with_safari/)
- regarding keyboard shortcuts, the only thing I found was that, to switch between tab groups you can use Command Shift Up/Down after the sidebar has the focus.
- In MacOS Ventura you can "[Choose a Tab Group for a Focus on Mac](https://support.apple.com/en-ca/guide/safari/ibrw36c4d917/mac)"
